product/roundhouse.console/Program.cs

```diff
@@ -300,6 +298,9 @@ private static void parse_arguments_and_set_up_configuration(ConfigurationProper
                 .Add("searchallinsteadoftraverse=|searchallsubdirectoriesinsteadoftraverse=",
                      "SearchAllSubdirectoriesInsteadOfTraverse - Each Migration folder's subdirectories are traversed by default. This option pulls back scripts from the main directory and all subdirectories at once. Defaults to 'false'",
                      option => configuration.SearchAllSubdirectoriesInsteadOfTraverse = option != null)
+                .Add("init",
+                     "Initialize",
+                     option => configuration.Initialize = option != null)

```

Comment:
Hmm, without duplicating configuration - this might end up being tricky. :) about to push up again.